blazar/plugins/oshosts/host_plugin.py

```diff
@@ -17,6 +17,7 @@
 import datetime
 import shlex
 import subprocess
+from random import shuffle
```

Comment:
This line should go above `import shlex` to pass pep8.

Reply:
Reformatted as requested.